### DISCUSSION TITLE
no-multiple-empty-lines

### DISCUSSION THREAD

**Discussion by jeancatarina** [biomejs/biome#1608] [Ideas]
### What happened?

hello, how can i make no-multiple-empty-lines work? because a I have a lot of empty lines in my project and the linter its not working

### Expected result

remove multiple lines empty

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by Conaclos** [comment]
This rule doesn't exist at the moment. I don't understand what you mean.

**Comment by ronnostic** [comment]
I'm coming from eslint and was surprised that it wasn't included.  This would be very nice to have!

**Reply by jeancatarina** [reply]
yes, this is a life changer, its hard for me to leave eslint without this rule

**Reply by Conaclos** [reply]
The formatter should make you happy?

**Comment by unvalley** [comment]
This is style related lint rule and eslint will remove (port to stylistic) it, so IMO we should handle this task by formatter in biome.

**Reply by tylerbutler** [reply]
The formatter is where I would expect this behavior to be found. I came across this discussion because I found some cases where `biome format` removes empty lines where [prettier leaves them as-is or condenses them to one](https://prettier.io/docs/en/rationale.html#empty-lines). Here's a playground link demonstrating the difference:

https://biomejs.dev/playground/?code=ZQB4AHAAbwByAHQAIAB7AAoACQBhACwACgAJAGIALAAKAAoACgAKACAAIAAgACAALwAvACAARABlAHQAYQBpAGwAcwAgAGEAYgBvAHUAdAAgAHQAaABlACAAZQB4AHAAbwByAHQAcwAgAGIAZQBsAG8AdwAKACAAIAAgACAAYwAsAAoAIAAgACAAIABkACwACgAKACAAIAAgACAALwAvACAAbQBvAHIAZQAgAGMAbwBtAG0AZQBuAHQAcwAKACAAIAAgACAAZQAsAAoAIAAgACAAIABmACwACgB9ACAAZgByAG8AbQAgACIALgAvAGYAbwBvAC4AagBzACIACgA%3D

**Comment by tiagoluizpoli** [comment]
Hi!

Is this being considered?

For me atleast, even though is a simple thing, is also definately a key feature. 

Brings so much clearness to the code (withoud manual intervention)

**Comment by nomagick** [comment]
Because Prettier intentionally dictates only one empty line in code, it has been on my ban list for years. I’d rather use the default formatter that comes with VSCode.

Today, I came across Biome. Just to let you know, for the same reason, I’d prefer to not bother moving away from VSCode default.

**Comment by nabato** [comment]
People have been using various empty spacing rules as visual designators for text depending on is it a header or a paragraph or a warning, etc. since the invention of writing. Only programmer's mind could come up with an idea to make text navigation harder in the name of uniformity.

**Comment by shamseerahammedm** [comment]
big massive +1, coming from eslint and standard land
